src/connectors/range/__tests__/connectRange-test.ts

```diff
@@ -1941,6 +1942,54 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/range-input
       });
     });
 
+    test('returns the `SearchParameters` with only one min and one max', () => {
```

Comment:
Considering https://github.com/algolia/instantsearch.js/pull/4627#issuecomment-770747015, the description should maybe be something like `'returns the SearchParameters with the correct price range'` or something more specific

Reply:
sounds good https://github.com/algolia/instantsearch.js/pull/4627/commits/b9bc2f33163a4bbcad72ff861e7cd649147f1881